### PR TITLE
feat: kibana setup for journey tracking

### PR DESCRIPTION
**Pull request by StefanWallin**
Status: Reviewable

### CODE REVIEW

File: packages/simulator/lib/statistics.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const { from } = require('rxjs')
+const { subscribe } = require('rxjs/operators')
+const { save } = require('./elastic')
+
+// TODO: Remove exampleJourney once the stream feeds this function instead
+const exampleJourney = {
+  id: 'vGLr-vxBkx',
```
Comment (StefanWallin) [inline packages/simulator/lib/statistics.js:7]: Vad tycker ni om denna data-struktur för Journeys vi vill mäta i Kibana? Works?

File: packages/simulator/lib/statistics.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  tripTime: 3000000,
+  weight: 0.041350018304702196,
+  pickup: { position: { lon: 11.016601, lat: 52.622044 } },
+  destination: { position: { lon: 22.20803, lat: 65.5825 } },
+  finalDestination: {
+    name: 'Lillstrandsvägen 2B',
+    position: { lon: 24.229152, lat: 66.589606 },
+  },
+  origin: 'Länstrafiken Norrbotten',
+  position: { lon: 13.016601, lat: 55.622044 },
```
Comment (StefanWallin) [inline packages/simulator/lib/statistics.js:21]: Strike position

File: packages/simulator/lib/statistics.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  id: 'vGLr-vxBkx',
+  status: 'Assigned',
+  co2: 0,
+  cost: 0,
+  distance: 0,
+  tripTime: 3000000,
+  weight: 0.041350018304702196,
+  pickup: { position: { lon: 11.016601, lat: 52.622044 } },
+  destination: { position: { lon: 22.20803, lat: 65.5825 } },
+  finalDestination: {
```
Comment (StefanWallin) [inline packages/simulator/lib/statistics.js:16]: Remove finaldestination and have only finaldestination as destination

Comment (Graborg) [inline packages/simulator/lib/statistics.js:16]: This should be removed, since it's redundant

File: packages/simulator/lib/statistics.js (hunk 2 above)
Comment (Graborg) [inline packages/simulator/lib/statistics.js:21]: This should be removed, since it's redundant

File: packages/simulator/index.js [hunk 4]
```diff
@@ -8,6 +8,12 @@ const regions = require('./streams/regions')
 const kommuner = require('./streams/kommuner')
 const { safeId } = require('./lib/id')
 
+const { save } = require('./lib/elastic')
+const statsCollector = require('./lib/statistics')
+from(
+  [statsCollector.exampleJourney] // TODO: Hook into stream
```
Comment (StefanWallin) [inline packages/simulator/index.js:14]: TODO!

**Pull request closed without merging**